### PR TITLE
Abandoning voltha-2.1 branch

### PR DESCRIPTION
**Pull request by andybavier**

### CODE REVIEW

File: minimal-values.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ images:
     # there is a need to specify the exact image repository and image tag.
     # If you would like to use the "master", "voltha-2.1", or other image
     # just replace the "tag" value below.
-    tag: voltha-2.1
+    tag: 2.1.0
```
Comment (davidkbainbridge) [inline minimal-values.yaml]: should this be a specific tag, or master

File: minimal-values.yaml [hunk 2]
```diff
@@ -20,8 +20,8 @@
 # set "<VALUE>" to "master". Similarly, if you want to use the images
 # generated from the HEAD of the "voltha-2.1" git branches, then uncomment
 # this block and replace "<VALUE>" with "voltha-2.1".
-defaults:
-  image_tag: voltha-2.1
+#defaults:
+#  image_tag: voltha-2.1
```
Comment (davidkbainbridge) [inline minimal-values.yaml]: kind-voltha typically goes beyond the latest tags or defaults in helm, so i think the default should now be master

File: restart-api.sh [hunk 3]
```diff
@@ -23,20 +23,20 @@ else
...
-echo "Scaling up voltha-api-server and ofagent"
-(set -x; kubectl scale --replicas=1 deployment -n voltha voltha-api-server ofagent)
+echo "Scaling up voltha-api-server and voltha-ofagent"
+(set -x; kubectl scale --replicas=1 deployment -n voltha voltha-api-server voltha-ofagent)
 echo "Waiting for PODS to be running"
 echo -n "Waiting "
 until [ $(kubectl -n voltha get pods | grep voltha-api-server | grep -c "2/2") -eq 1 ]; do echo -n "."; sleep 3; done
-until [ $(kubectl -n voltha get pods | grep ofagent | grep -c "1/1") -eq 1 ]; do echo -n "."; sleep 3; done
+until [ $(kubectl -n voltha get pods | grep voltha-ofagent | grep -c "1/1") -eq 1 ]; do echo -n "."; sleep 3; done
 echo " Running"
```
Comment (davidkbainbridge) [inline restart-api.sh]: as this restart is no longer needed to support a work around for a voltha bug, what do you think about just removing it?

File: minimal-values.yaml (hunk 1 above)
Comment (andybavier) [reply minimal-values.yaml]: I was thinking that, by default, kind-voltha should use released images.  If you want to use master then it's easy to override.

File: minimal-values.yaml (hunk 2 above)
Comment (andybavier) [reply minimal-values.yaml]: I was thinking that, by default, kind-voltha should use released images (i.e., the ones specified in the charts).  If you want to use master then it's easy to override.

File: restart-api.sh (hunk 3 above)
Comment (andybavier) [reply restart-api.sh]: That's fine with me

File: minimal-values.yaml (hunk 2 above)
Comment (dbainbri-ciena) [reply minimal-values.yaml]: kind-voltha has always overridden the default helm versions (which are the released versions) with the `HEAD` of a branch. Now that we are moving to only a single branch I think the default needs to be `master`

File: minimal-values.yaml (hunk 1 above)
Comment (dbainbri-ciena) [reply minimal-values.yaml]: kind-voltha has always overridden the default helm versions (which are the released versions) with the HEAD of a branch. Now that we are moving to only a single branch I think the default needs to be master

Comment (andybavier) [reply minimal-values.yaml]: That's fine for the voltha/voltha-onos image

**Comment by andybavier** [comment]
I've revised the pull request to change the default tags to :master.  Agreed that the common case is likely pulling :master images.  This makes it slightly more difficult to use kind-voltha to install using the released images in the charts, since AFAIK Helm doesn't allow you to --unset a variable, but it's not a huge deal.

**Pull request merged**